ci.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python2
```

Comment:
On Mac OSX this generates:
/usr/bin/python2: bad interpreter: No such file or directory
(sadly)